README.md

```diff
@@ -809,12 +809,10 @@ provides the schema when `is` resolves truthy, and `otherwise` provides the sche
 ##### Available options for providing `is`
 
 When `is` is not provided (`=== undefined`) it is strictly resolved as `Boolean(value)` wherein `value` is the current
-value of the referenced sibling. Note that if the refeferenced sibling is an array that this will mean the result is
-always `true` because `Boolean([...])` is true.
+value of the referenced sibling. Note that if multiple siblings are referenced then all the values of the array need to
+resolve truthy for the `is` to resolve truthy.
 
-When `is` is a primitive literal it is strictly compared (`===`) to the current value of the referenced sibling, or
-through [`Array#some`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) with
-a strict negative comparison (`!==`) for each element if multiple siblings are referenced.
+When `is` is a primitive literal it is strictly compared (`===`) to the current value.
```

Comment:
Why did you delete the blurb about using `Array#some`?

Reply:
2nd point doesn't need it now. 1st point does use it so I just added what happens when the keys are an array. I can add this `some` part to the first point too if u want